docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ConvertTimezone
+---
+
+# ConvertTimezone
```

Comment:
I'm not very good at wording but here is a diagram of how we process convertTimezone depending on the input.

![Screenshot 2022-11-25 at 17 36 48](https://user-images.githubusercontent.com/25661381/203964971-89eb8539-c500-48a1-ba04-b463c1e71125.png)

Reply:
[convert_timezone.zip](https://github.com/metabase/metabase/files/10091711/convert_timezone.zip)
just drop this into draw.io if you want to view the diagram